src/main/java/org/trips/service_framework/events/SnsEventPublisher.java

```diff
@@ -0,0 +1,42 @@
+package org.trips.service_framework.events;
+
+import com.amazonaws.services.sns.model.PublishRequest;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.trips.service_framework.configs.SnsConfig;
+import org.trips.service_framework.events.dto.SkuCreationTopicMessage;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class SnsEventPublisher {
```

Comment:
This will hardwire SNS with the framework, can we explore a way where SKU creation is opt-in? @hardikph @cf-aagam